pvlib/tests/test_atmosphere.py

```diff
@@ -82,12 +82,50 @@ def test_gueymard94_pw():
     pws = atmosphere.gueymard94_pw(temps_humids[:, 0], temps_humids[:, 1])
 
     expected = np.array(
-        [  0.1       ,   0.33702061,   1.12340202,   0.1       ,
+        [0.1       ,   0.33702061,   1.12340202,   0.1       ,
          1.12040963,   3.73469877,   0.1       ,   3.44859767,  11.49532557])
 
     assert_allclose(pws, expected, atol=0.01)
 
 
+# Unit tests
+def test_rh_from_tdew(
+    tdew_rh_conversion_temperature, tdew_rh_conversion_dewpoint,
+    tdew_rh_conversion_relative_humidity
+):
+
+    # Calculate relative humidity
+    rh = atmosphere.rh_from_tdew(
+        temperature=tdew_rh_conversion_temperature,
+        dewpoint=tdew_rh_conversion_dewpoint
+    )
+
+    # test
+    pd.testing.assert_series_equal(
+        rh,
+        tdew_rh_conversion_relative_humidity,
+        check_names=False
+    )
```

Comment:
For completeness, let's add corresponding tests where the inputs are float and array.  We will also need to include tests where alternative coefficient values are specified.

Reply:
@kandersolar just reworked the unit tests, I added tests which take in a numpy array and a float as well as a pandas series with a different set of coefficients.  Let me know what you think, happy to adjust in any way.